src/DynamoWebServer/Messages/MessageHandler.cs

```diff
@@ -176,12 +176,9 @@ private WorkspaceModel GetWorkspaceByGuid(string guidStr)
         {
             Guid guidValue;
             if (!Guid.TryParse(guidStr, out guidValue) || guidValue.Equals(Guid.Empty))
-                return dynamoModel.HomeSpace;
+                return dynamoModel.Workspaces.First(w => w is HomeWorkspaceModel);
```

Comment:
there is no `HomeSpace` in dynamo model anymore, so I choose it from the workspaces list